src/palette/internal/palettecreator.cpp

```diff
@@ -828,7 +828,7 @@ PalettePtr PaletteCreator::newOrnamentsPalette(bool defaultPalette)
         auto ornament = Factory::makeOrnament(gpaletteScore->dummy()->chord());
         ornament->setSymId(ornamentType);
         qreal mag = ornament->symId() == SymId::ornamentTrill ? 1.0 : 1.2;
-        sp->appendElement(ornament, ornament->typeUserName(), mag);
+        sp->appendElement(ornament, SymNames::userNameForSymId(ornamentType), mag);
```

Comment:
I'm not exactly sure why this one is necessary, since `Ornament::typeUserName()`, which is actually `Articulation::typeUserName()`, should produce exactly the same result. But if this is really necessary, then it's fine I guess

Reply:
Couldn't get it to work without, and a similar mechanism is used in other places.
I'll add it to articulation to, to fix that one oddnes